app/assets/stylesheets/application.scss

```diff
@@ -8,6 +8,9 @@ $govuk-new-link-styles: true;
 $govuk-include-default-font-face: false;
 
 @import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/components/intervention";
+
+@import "components/intervention";
```

Comment:
We can remove this import by using the spacing classes from the design system, see further info below.